libcomposefs/lcfs-writer.c

```diff
@@ -275,9 +275,9 @@ int lcfs_write(struct lcfs_ctx_s *ctx, void *_data, size_t data_len)
 
 	if (ctx->write_cb) {
 		while (data_len > 0) {
+			errno = EIO;
```

Comment:
Hmm, I guess we need to do this because we can't rely on callers setting errno...and indeed in our own code which uses `fwrite`, it doesn't?

So...okay, is there *really* no way to get the actual error out of `ferror`?

Reply:
Oh, I guess `fwrite` is...somewhat expected conventionally to set `errno`, but it's not standardized?